_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png
![68747470733a2f2f6f63746f6465782e6769746875622e636f6d2f696d616765732f79616b746f6361742e706e67](https://user-images.githubusercontent.com/56514617/178783369-245d2be9-cf37-4476-aaf4-b11ed908c022.png)
)